### PR TITLE
Create December 2018 Gazette

### PR DESCRIPTION
**Pull request by JBHyperion**
---

### CODE REVIEW

File: news/2018-12-23-the-qat-gazette-december-2018.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+## Summary
+
+This brings us to the end of this month’s news, (almost) the end of the year, but sadly, also the end of my tenure as editor for the QAT Gazette. You may have noticed that we recently opened a [Twitter account](https://twitter.com/osuqat), bringing you all the glorious news you know and love from the Gazette, but in a more regular, bite-sized fashion. And also cute cat pictures if that’s more your speed. Please follow us if you’d like to continue receiving all the latest and greatest updates from the osu! mapping and modding community!
+
+Contributing to the Gazette as both a writer and editor over the past two years has been a great pleasure, and I will sorely miss being able to communicate with you all as we share occasions both exciting and intriguing, happy and sad. I have loved watching on as osu! and its community has continued to grow, evolve, and achieve truly great things together over my time here, and I sincerely hope that 2019 brings more of the same!
+
+To all a Merry Christmas, and a Happy New Year!
+
+**–JBHyperion**
```
Comment (Joehuu) [inline news/2018-12-23-the-qat-gazette-december-2018.md]:
```suggestion
–JBHyperion
```

File: news/2018-12-23-the-qat-gazette-december-2018.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+ 
+All mentioned members will receive a shiny new profile badge (or incremental flair upgrade if they earned one previously), which we encourage them to wear with pride in appreciation of their meritorious voluntary service.
+
+### New Beatmap Nominator Promotions
+
+With the potential for significant change coming to the QAT over the coming weeks (as will be discussed later in this article), there will naturally be an uncertainty for future Beatmap Nomintaor applications. Regular scheduling would have the next cycle start in mid- to late-January, but at present, the team is debating whether these can (or should) be operated alongside the QAT restructure. We will bring you a more clear update on this matter once it has been decided upon.
```
[the remark this answers is not in the document]
Comment (JBHyperion) [reply news/2018-12-23-the-qat-gazette-december-2018.md]: de-typo'd

File: news/2018-12-23-the-qat-gazette-december-2018.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+### Beatmap Nominator Retirements
+
+With the prospect of new Nominator applications to hopefully be announced soon, we must also take a moment to acknowledge the lasting contributions of those who have since moved on. Please show your appreciation for [hikiko-](https://osu.ppy.sh/users/6512678), [Wishkey](https://osu.ppy.sh/users/1101704) and [Yauxo](https://osu.ppy.sh/users/898306) – may their future endeavours be as prosperous as their time in osu!
```
[the remark this answers is not in the document]
Comment (JBHyperion) [reply news/2018-12-23-the-qat-gazette-december-2018.md]: Added fullstop

File: news/2018-12-23-the-qat-gazette-december-2018.md (hunk 1 above)
Comment (JBHyperion) [reply news/2018-12-23-the-qat-gazette-december-2018.md]: rip bold

File: news/2018-12-28-the-qat-gazette-december-2018.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+### Rules Regarding Mapping Commissions
+
+Just prior to the proposed Quality Assurance Team changes, significant discussion amongst the osu!team and QAT took place to clarify our position on paid-for mapping and modding contributions, as well as how we should handle such cases going forward. Whilst such mapping and modding commissions are certainly not a new appearance in osu!, they have gained significant traction and mainstream attention recently. Coupled with osu!’s awesome and ever-expanding library of licensed *Featured Artists*, it was decided to tidy up some potentially grey areas.
+ 
+You can read an [announcement from the osu!team](https://osu.ppy.sh/community/forums/topics/840838) on the topic for a detailed outlook, but in brief summary, commissions will be permitted to a certain extent, provided they are undertaken tastefully and the users profiting from them do not occupy a position within the Beatmap Nominators, Quality Assurance or other group within the osu!team. With this move, we wish to both emphasize osu!’s core “free-to-win” value, and promote a positive image and reputation with our current and potential future Featured Artists.
```
Comment (Xinnoh) [inline news/2018-12-28-the-qat-gazette-december-2018.md]: or other group -> or other groups

Comment (JBHyperion) [reply news/2018-12-28-the-qat-gazette-december-2018.md]: Either is fine, people can be in one group or multiple

File: news/2018-12-28-the-qat-gazette-december-2018.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+Behold, the Christmas gift of mapping and modding news for all! With major changes to the Quality Assurance Team on the horizon, what will the final days of 2018 bring as a new dawn prepares to rise?
+
+![](/wiki/shared/news/banners/qat-blog-logo.png)
+
+## Opening
+
+by [JBHyperion](https://osu.ppy.sh/users/4879508)
+
+Greetings everyone – it’s been a while!
```
[the remark this answers is not in the document]
Comment (JBHyperion) [reply news/2018-12-28-the-qat-gazette-december-2018.md]: Fixed throughout

File: news/2018-12-28-the-qat-gazette-december-2018.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+Many eagle-eyed readers may have noticed a distinct absence of October and November’s articles, for which I must apologise – I was stricken by a mysterious and fickle beast during my travels, one spoken of in hushed whispers and frightening children’s tales, known only as the *Real Life*… Truly terrifying indeed…
```
[the remark this answers is not in the document]
Comment (JBHyperion) [reply news/2018-12-28-the-qat-gazette-december-2018.md]: Fixed quotes and dots

File: news/2018-12-28-the-qat-gazette-december-2018.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+
+## Within the Beatmap Nominators
+
+With our newest wave of Beatmap Nominators having recently received their first evaluation results, as well as a new round of applications slated for the New Year, it’s full steam ahead as usual! However, we must also acknowledge the contributions of our long-standing members, some of whom have sadly decided to take a well-earned break. Whichever news you seek, it’s all right here for you!
+
+### Changes to Nominator Evaluations
+
+The Beatmap Nominator Probation system has facilitated a rapid expansion of the team since its inception, offering active modding community members a chance to prove themselves in a more controlled environment. However, we understand that in some cases, feedback on each Nominator’s performance may either cause members to focus on “safer” nominations in order to ensure a more favourable evaluation, or come too late to allow for reasonable improvement.
```
[the remark this answers is not in the document]
Comment (JBHyperion) [reply news/2018-12-28-the-qat-gazette-december-2018.md]: Fixed speech quotes also

File: news/2018-12-29-the-qat-gazette-december-2018.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+
+### Proposed changes to the QAT
+
+As mentioned several times in this article already, significant changes could be just on the horizon for the Quality Assurance Team, which may drastically change how we operate in the future. You can find more details on this announcement and the reasons for it in the Development forum, where [discussions towards multiple osu!team and community proposals](https://osu.ppy.sh/community/forums/topics/842601) are currently taking place.
+
+The first of these changes has been one of leadership, with [-Mo-](https://osu.ppy.sh/u/2202163) replacing myself as QAT Leader, alongside the continuing [Mao](https://osu.ppy.sh/users/2204515). Whilst I have thoroughly enjoyed my time at the head of the QAT, my free time available for osu! has been reduced considerably over the past few months, and I no longer feel I can properly fulfil the expectations of my role, or the community who elected me to it. On a positive note, new faces provide the potential for new direction, so please support -Mo- warmly as he settles into the challenge of management!
```
Comment (WalterToro) [inline news/2018-12-29-the-qat-gazette-december-2018.md]: -Mo- link should use /users/ instead of /u/

Comment (JBHyperion) [reply news/2018-12-29-the-qat-gazette-december-2018.md]: Whoops, fixed

**Pull request merged**